### DISCUSSION TITLE
RCS integration

### DISCUSSION THREAD

**Discussion by heroic** [chatwoot/chatwoot#7922] [Feature Requests]
RCS is picking up as a great way to message Android users and as such it is seeing traction in replacing whatsapp messaging. There is currently no plan to enable RCS in chatwoot as far as I can tell. Is there a possibility to add this?

**Comment by mahaddad** [comment]
second this. Or how to tell what will and will not be supported from RCS in chatwoot if I am using Twilio messaging service?

**Comment by Realmullens** [comment]
We need this ASAP. Especially with Twilio's new RCS feature.

**Comment by ldaniel-jmz** [comment]
+1

**Comment by devangn** [comment]
+1 please

**Comment by edgain** [comment]
+1

**Comment by calebjacobsonns** [comment]
+1

**Comment by ldaniel-jmz** [comment]
Now that Meta is restricting the use of AI bots on WhatsApp (via new terms), RCS and Google have a clear path. So this RCS functionality becomes more relevant than ever.

**Reply by ldaniel-jmz** [reply]
Hope chatwoot team consider this 🙌🙏! @sojan-official  @pranavrajs

**Reply by qentri** [reply]
Indeed, this is the opportunity for RCS to take over! The complete idea on this article: [Meta is starting to restrict AI Bots on WhatsApp. This is Google RCS's opportunity to strike.](https://www.reddit.com/r/UniversalProfile/comments/1oikcqw/meta_is_starting_to_restrict_ai_bots_on_whatsapp/)

And therefore Chatwoot should implement it.

**Reply by sarahmanning190** [reply]
@sojan-official @pranavrajs Do you have this on your pipeline?
